prog/vm/github_runner.rb

```diff
@@ -52,7 +52,8 @@ def pick_vm
       enable_ip4: true,
       arch: label_data["arch"],
       allow_only_ssh: true,
-      swap_size_bytes: 4294963200 # ~4096MB, the same value with GitHub hosted runners
+      swap_size_bytes: 4294963200, # ~4096MB, the same value with GitHub hosted runners
+      force_new_subnet: true
```

Comment:
Instead of introducing a new `force_new_subnet` parameter, what do you think about just assemble a new PS here and pass `private_subnet_id`. I think the runners are the only use case.

Reply:
Updated.